lib/model.js

```diff
@@ -40,6 +40,10 @@ var Utils = require('./utils')
  */
 class Model {
 
+  static get Instance() {
+    return this;
+  }
```

Comment:
Leave it for now, let's make the goal to get this PR merged asap so we dont have to rebase.
And then work on removing all cruft etc.